docs/App.UiLib.md

```diff
@@ -13,6 +13,8 @@ To that end, we are going to use [react-native-ui-lib](https://github.com/wix/re
 
 It was designed to enable developers to build and customize their own visual language so it is not specific in any way to Wix. The Wix mobile app, for example, is using a private UI library, wix-react-native-ui-lib, as a wrapper around react-native-ui-lib. Most of the components come directly from react-native-ui-lib and are customized to fit the purpose and visual language of the Wix mobile app.
 
+Please read more in our uilib's [Wiki Page](https://github.com/wix-private/wix-react-native-ui-lib/wiki) (see table of content)
```

Comment:
This is the open source library so we can't add it in here. The plan is to have 2 separate libraries public and private. This article is only related to the open source.

Reply:
oy, my bad. 
can we revert?